### PR TITLE
261: Add Kafka To Master Chart

### PR DESCRIPTION
**Pull request by bmitchinson**
- Updated forklift + valkyrie to use the newly placed kafka broker
- Added reasonable default resources for forklift to ensure it could run
- Upgraded kafka to `2.5.1` and strimzi to `22.0`

### CODE REVIEW

**Comment by srashee1** [comment]
Do you need to add kafka to the requirements.yaml for urbanOS?

**Comment by srashee1** [comment]
> Do you need to add kafka to the requirements.yaml for urbanOS?

I just realized you can't release without first doing this... Right?

File: kafka/templates/topics.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{- range $topic := .Values.kafka.topics }}
+---
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: {{ $topic.name }}
+  namespace: {{ $topic.namespace | default $.Release.Namespace }}
```
Comment (ksmith-accenture) [inline kafka/templates/topics.yaml:7]: I think the namespace should always be the release namespace unless we come across a future use case for putting the topic in another namespace.  This implies that we support a multi-namespaced deployments in a way that we do not

**Review by ksmith-accenture — CHANGES_REQUESTED** [review]

**Comment by bmitchinson** [comment]
> > Do you need to add kafka to the requirements.yaml for urbanOS?
> 
> I just realized you can't release without first doing this... Right?

Kinda. Good catch!

This is part one of two, (which I should have explained in the body of the PR, so my apologies). In order to update UrbanOS requirements, the kafka chart / tar needs to exist on the `/charts` remote. So the plan is to merge this (the dependency changes) and then have an immediate follow up that updates the UrbanOS chart to utilize those previously altered dependancies.

File: kafka/templates/topics.yaml (hunk 1 above)
Comment (bmitchinson) [reply kafka/templates/topics.yaml]: Good idea, this was changed 👍🏼

**Review by ksmith-accenture — APPROVED** [review]

**Pull request merged**